ynr/apps/candidates/models/popolo_extra.py

```diff
@@ -242,6 +242,15 @@ def has_results(self):
             return True
         return False
 
+    @property
+    def uncontested(self):
+        if (
+            self.get_winner_count == self.post.memberships.count()
```

Comment:
I think this needs to be `self.membership_set.count()` as you only want to compare against memberships (candidates) for this specific ballot - rather than `post.memberships.count()` which will return the number of candidates that have stood for the post across _all_ ballots (e.g. all previous years).

As an aside, for an explanation as to why the reverse lookup is `membership_set` on Ballot, but `memberships` on Post, have a look at [this section of the docs](https://docs.djangoproject.com/en/3.2/topics/db/queries/#following-relationships-backward). 

Also you can save a DB query for ballots that are not locked by breaking up the conditional and returning early:
```
if not self.candidates_locked:
    return False

return self.get_winner_count == self.membership_set.count()
```